Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -109,34 +109,30 @@ static inline void collectLayoutAttributes(RenderObject* text, Vector<SVGTextLay
     }
 }
 
-static inline bool findPreviousAndNextAttributes(RenderElement& start, RenderSVGInlineText* locateElement, bool& stopAfterNext, SVGTextLayoutAttributes*& previous, SVGTextLayoutAttributes*& next)
+static inline bool findPreviousAndNextAttributes(RenderElement& root, RenderSVGInlineText* locateElement, SVGTextLayoutAttributes*& previous, SVGTextLayoutAttributes*& next)
```

Comment:
The callers of this function do not check the return bool value. They have also to define and initialize the arguments `previous` and `next`. How about this prototype:
```
static inline std::tuple<SVGTextLayoutAttributes*, SVGTextLayoutAttributes*> findPreviousAndNextAttributes(RenderElement& root, RenderSVGInlineText& locateElement)
```
And the callers can just call it this way:
```
auto [previous, next] = findPreviousAndNextAttributes(*this, attributes->context());
```